_usage/command-line.md

```diff
@@ -1,212 +1,121 @@
 ---
 layout: default
-title: Using the command line
+title: Connecting to Babelfish
 nav_order: 1
 ---
 
-## Using client tools to work with Babelfish
+## Using client tools to connect to Babelfish
 
-Most people use SQL Server Management Studio (SSMS) on Microsoft Windows to
-connect to Babelfish.
-
-However, there is an ever-growing community which prefers to use the command
-line to work with the server.
-Those users have the following command-line tool choices on Linux:
+On Windows, most people use SQL Server Management Studio (SSMS) or `sqlcmd` to
+connect to SQL Server, and these can be used to connect to Babelfish as well.
+SSMS is not available on Linux, so you are limited to using `sqlcmd` there.
 
-- sqlcmd: Using the Microsoft SQL Server interface
-- tsql: Using the FreeTDS command line tool
+Other clients that speak the TDS protocol can also be used, but are not
+officially supported by Babelfish.  On Linux, these include the FreeTDS command
+line tool `tsql`.
 
-Note that there is also a `psql` version for Babelfish, but that is not reliable.
-Therefore, it is strongly recommended not to use it.
+Using the PostgreSQL protocol, you can use `psql` or any other client to
+connect to Babelfish.
 
 
-### Using sqlcmd to connect to the DB cluster {#babelfish-connect-sqlcmd}
+### Using `sqlcmd` to connect to the DB instance {#babelfish-connect-sqlcmd}
 
 The most common way to connect to and interact with Babelfish
-is with the SQL Server <code>sqlcmd</code> utility, as shown in the following.
+is with the SQL Server `sqlcmd` utility, as shown in the following.
+
 
 ```bash
 sqlcmd -S host.sample.com,1433 -U PUT_USER_HERE -P PUT_PASSWORD_HERE -d PUT_DBNAME_HERE
 ```
 
 Where:
 
-- -S is the endpoint and port of the DB cluster.
-- -U is the login name of the user.
-- -P is the password associated with the user.
-- -d is the name of your Babelfish database.
+- `-S` is the server name and port of the DB instance.
+- `-U` is the login name of the user.
+- `-P` is the password associated with the user.
+- `-d` is the database that is selected initially.
 
+Use the `USE` command to select a database, and don't forget to enter `GO`
+after entering one or more SQL statements to send them to the server.
```

Comment:
How about:

After connecting with `sqlcmd`, you can use familiar SQL Server syntax to create and manage database objects.

Reply:
Done; I agree that is better.